ui/client/components/graph/Graph.js

```diff
@@ -40,16 +41,16 @@ class Graph extends React.Component {
     })
     this.nodesMoving();
 
-    this.espGraphRef = React.createRef()
-    this.parent = document.getElementById(this.props.parent)
+    this.myRef = React.createRef()
```

Comment:
MyRef? Mayby better will be graphRef?

Reply:
Sure, its work in progress, I'll clean it up.